spa/views.py

```diff
@@ -51,8 +52,12 @@ def spa_view(request):
         res.set_cookie("jwt_token", token, httponly=True, secure=False)
     return res
 
-
 def spa_view_catchall(request, catchall):
```

Comment:
it is better to create a new view.
catchall if for random url that are not registerd